content/applications/sales/subscriptions/ecommerce.rst

```diff
@@ -0,0 +1,48 @@
+=======================================
+Use subscriptions in the eCommerce shop
+=======================================
+
+Subscription products can be sold in the Odoo *eCommerce* shop just like regular sales products.
+
+However, by default, the eCommerce product page will only display the shortest recurrence period
+listed in the :guilabel:`Time-based pricing` tab of the product form. For example, if a
+subscription product has Monthly and Yearly recurrence periods configured, then only the Monthly
+price will appear on the eCommerce page for that product.
+
+To add more recurrence periods to the eCommerce product page, create the different recurrence
+periods as *product variants*.
```

Comment:
It might be good to link to the documentation for creating subscription products and product variants since some of the configuration options below don't appear by default.